signac/contrib/project.py

```diff
@@ -89,15 +89,16 @@ def __setitem__(self, key, value):
 class Project:
     """The handle on a signac project.
 
-    Application developers should usually not need to
-    directly instantiate this class, but use
-    :meth:`~signac.get_project` instead.
+    A :class:`Project` may only be constructed in a directory that is already a
+    signac project, i.e. a directory in which :func:`~signac.init_project` has
+    already been run. If project discovery (searching upwards in the folder
+    hierarchy until a project root is discovered) is desired, users should
+    instead invoke :func:`~signac.get_project` or :meth:`Project.get_project`.
 
     Parameters
     ----------
-    config :
-        The project configuration to use. By default, it loads the first signac
-        project configuration found while searching upward from the current
+    root : str, optional
+        The project root directory. By default, the current working directory
```

Comment:
```suggestion
        The project root directory. By default, the current
```
Is this repeated in line 102?

Reply:
Ha good catch thanks.